swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md

```diff
@@ -167,3 +167,21 @@ To reverse the deprecation of a package, execute the same command with the `--un
 ```
 $ bal deprecate <org-name>/<package-name>:<version> --undo
 ```
+
+### Deprecate all the versions of a published package
+
+To change the name of a published package, you can mark all the versions of the existing package as deprecated and then publish the package under a new name.
+
+The package owner can run the following command. An optional deprecation message can also be included.
```

Comment:
```suggestion
The package owner can run the following command. An optional deprecation message can also be included.
```
Shall we add an info or tip section on this?

Reply:
Fixed.